### PR TITLE
Inline langchain-databricks components

### PR DESCRIPTION
**Pull request by prithvikannan**
Inline ChatDatabricks, DatabricksEmbeddings, and DatabricksVectorSearch from `langchain-databricks` package and their corresponding unit tests.

Remove `langchain-databricks` as a dependency for `databricks-langchain`.

### CODE REVIEW

**Comment by leonbi100** [comment]
@prithvikannan are we considering porting over the integration tests? I understand that's more complicated since they have their own CI/CD system

File: integrations/langchain/src/databricks_langchain/__init__.py [hunk 1]
```diff
@@ -1,11 +1,8 @@
 # Import modules from langchain-databricks
```
Comment (leonbi100) [inline integrations/langchain/src/databricks_langchain/__init__.py]: Are we going to remove `langchain-databricks` as a dependency from the `pyproject.toml` file?

**Comment by prithvikannan** [comment]
> @prithvikannan are we considering porting over the integration tests? I understand that's more complicated since they have their own CI/CD system

yes we do plan to. this PR is just the unit tests, but we will move integration tests next.

File: integrations/langchain/src/databricks_langchain/__init__.py (hunk 1 above)
Comment (prithvikannan) [reply integrations/langchain/src/databricks_langchain/__init__.py]: good callout, just updated.

Comment (leonbi100) [reply integrations/langchain/src/databricks_langchain/__init__.py]: I think they're some new test issues with that change could you take a look?

**Review by leonbi100 — APPROVED** [review]
lgtm!

**Pull request merged**